client/src/app/console/window.tpl.html

```diff
@@ -0,0 +1,14 @@
+<div class="panel panel-primary" ng-controller="ConsoleCtrl">
```

Comment:
das ng-controller-attribut auf den parent in index.html zu setzen würde nicht funktionieren?